start.sh

```diff
@@ -16,21 +17,48 @@ export COMPOSE_PROJECT_NAME=${COMPOSE_PROJECT_NAME:-e}
 export SCALE=${SCALE:-3}
 
 run() {
-    echo "Using discovery node for bootstrapping ${SCALE}-node cluster."
+  if [[ -n "${DOCKER_HOST}" ]]; then
```

Comment:
This will behave unexpectedly if you're running docker-machine (which sets a RFC1918 address) rather than Docker For Mac (which uses the local socket).